plugins/modules/homebrew.py

```diff
@@ -399,10 +399,14 @@ def _extract_package_name(self, package_detail, is_cask):
             name = package_detail["name"]
             full_name = package_detail["full_name"]
 
-        tapped_name = package_detail["tap"] + "/" + name
+        # Issue #9803: name can include the tap as a prefix, in order to
+        # disambiguate, e.g. casks from identically named formulae.
```

Comment:
How about adding the URL instead:
```suggestion
        # Issue https://github.com/ansible-collections/community.general/issues/10012:
        # name can include the tap as a prefix, in order to disambiguate,
        # e.g. casks from identically named formulae.
```

Reply:
yup, but the link to 10012 should go in line 405 below, not here

Reply:
Right. I've fixed this comment and added a suggestion for further below.